src/api/test-mocks/mock-application.ts

```diff
@@ -254,4 +254,11 @@ export const mockApplicationMultiplePolicy = {
   policy: mockMultiplePolicy,
 } as Application;
 
+export const mockApplicationCreationObject = {
```

Comment:
Worried about misalignment here, checkout `createInitialApplication`

I think we should extract the initial application data from here into somewhere, maybe constants - then can consume this instead of having a separate new mock here for "many applications"

Reply:
done